### PR TITLE
Option Base 0

### PR DESCRIPTION
**Pull request by Hosch250**

### CODE REVIEW

File: RetailCoder.VBE/Inspections/OptionBaseZeroInspection.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        }
+
+        public class OptionBaseStatementListener : VBAParserBaseListener
+        {
+            private readonly IList<VBAParser.OptionBaseStmtContext> _contexts = new List<VBAParser.OptionBaseStmtContext>();
+            public IEnumerable<VBAParser.OptionBaseStmtContext> Contexts { get { return _contexts; } }
+
+            public override void ExitOptionBaseStmt(VBAParser.OptionBaseStmtContext context)
+            {
+                if (context.numberLiteral()?.INTEGERLITERAL().Symbol.Text == "0")
```
Comment (comintern) [inline RetailCoder.VBE/Inspections/OptionBaseZeroInspection.cs]: Nope.  C# 6

File: RetailCoder.VBE/Inspections/Resources/InspectionsUI.resx [hunk 2]
```diff
@@ -650,4 +650,16 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="AssignedByValParameterMakeLocalCopyQuickFix" xml:space="preserve">
     <value>Create and use a local copy of the parameter</value>
   </data>
-</root>
+  <data name="OptionBaseZeroInspectionMeta" xml:space="preserve">
+    <value>'Option Base 0' is the default base</value>
```
Comment (retailcoder) [inline RetailCoder.VBE/Inspections/Resources/InspectionsUI.resx]: I would just make it "'Option Base 0' is redundant"

Comment (Hosch250) [inline RetailCoder.VBE/Inspections/Resources/InspectionsUI.resx]: Sure.

Comment (Hosch250) [inline RetailCoder.VBE/Inspections/Resources/InspectionsUI.resx]: That's the meta.

Comment (retailcoder) [inline RetailCoder.VBE/Inspections/Resources/InspectionsUI.resx]: > **'Option Base 0' is redundant**
> This is the default setting, it does not need to be specified.

**Pull request merged**